src/main/java/com/example/Restaurant_Management_System_REST_API/rest/controller/advice/GlobalCustomExceptionHandler.java

```diff
@@ -20,4 +21,10 @@ public ResponseEntity<String> handlePropertyValueException (PropertyValueExcepti
         return new ResponseEntity<>("Something is missing! You didn't specified a required field in your request: "
                 + ex.getMessage(), HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(ConstraintViolationException.class)
+    public ResponseEntity<?> handleConstraintViolationException (ConstraintViolationException ex) {
+        return new ResponseEntity<>("You haven't gone through all validations! See what is missing here: " +
+                ex.getMessage(), HttpStatus.BAD_REQUEST);
```

Comment:
Instead of 400 code, the 422 could be used:
https://stackoverflow.com/questions/16133923/400-vs-422-response-to-post-of-data

At least that what I'd do and imo it's a case here, too.

Reply:
OK, I see the point!